frontend/appflowy_flutter/lib/plugins/database/widgets/field/type_option_editor/select/select_option.dart

```diff
@@ -206,22 +206,23 @@ class CreateOptionTextField extends StatefulWidget {
 }
 
 class _CreateOptionTextFieldState extends State<CreateOptionTextField> {
-  late final FocusNode _focusNode;
+  final focusNode = FocusNode();
 
   @override
   void initState() {
     super.initState();
-    _focusNode = FocusNode()
-      ..addListener(() {
-        if (_focusNode.hasFocus) {
-          widget.popoverMutex?.close();
-        }
-      });
-    widget.popoverMutex?.listenOnPopoverChanged(() {
-      if (_focusNode.hasFocus) {
-        _focusNode.unfocus();
-      }
-    });
```

Comment:
@richardshiue Found a minor issue when refactoring the popover: if using an anonymous function here, then it can't be unregistered when disposing.